.github/CODEOWNERS.disabled

```diff
@@ -33,7 +33,6 @@
 # FOLDER-SPECIFIC REVIEWERS:
 
 /moveit_plugins/moveit_ros_control_interface/               @ipa-mdl @bmagyar
```

Comment:
Should we remove `moveit_ros_control_interface` .? we're not using it at all

Reply:
How about in a new PR?